### PR TITLE
Collaborations and Compatabilities

### PR DESCRIPTION
**Pull request by jonathf**
A recognition of the people and projects using chaospy and compatibilities with other projects.

This mostly updates the docs. Pre-releases can be found here:
* [README.rst](https://github.com/jonathf/chaospy/tree/development)
* [read-the-docs](https://chaospy.readthedocs.io/en/development/recipes/external.html)

### CODE REVIEW

**Comment by jonathf** [comment]
@krystophny, @regislebrun, @nschloe, @simetenn, I am updating the docs to include some shout-outs to related projects. Please let me know if you are happy with how you and your projects are portrait, or if you want something changed/hidden before I merge.

File: chaospy/external/openturns_.py [hunk 1]
```diff
@@ -0,0 +1,149 @@
...
+    def __init__(self, distribution):
+        from openturns import ComposedDistribution, ContinuousDistribution
+        if isinstance(distribution, ComposedDistribution):
+            distributions = distribution.getDistributionCollection()
+            if not numpy.allclose(
+                distribution.getLinearCorrelation(),
+                numpy.eye(len(distributions))
+            ):
+                raise DependencyError("Stochastically dependent "
+                                      "OpenTURNS distribution unsupported")
```
Comment (jonathf) [inline chaospy/external/openturns_.py]: @regislebrun, I hope you don't mind me asking:
Is there a more correct way to check for stochastic dependencies in `openturns` than linear correlation? I am starting to look into the internals of OpenTURNS, but it is a big project and I don't have the overview I'd like.

**Comment by jonathf** [comment]
@flo2k, before I add you as well, does your evacuation project have a webpage you want me to link to?

**Comment by flo2k** [comment]
Hi Jonathan,

Right now, there is only a private repository with my stuff. I think I'm able to give this to a broader audience after I have finished my thesis. The working title is at the moment UQEF (Uncertainty Quantification Execution Framwork).

Best,

Florian

**Comment by krystophny** [comment]
@ jonathf everything fine from my side - thanks!

**Comment by simetenn** [comment]
This looks good for me as well :+1:

File: chaospy/external/openturns_.py (hunk 1 above)
Comment (regislebrun) [reply chaospy/external/openturns_.py]: Hi,
When you build a multivariate distribution you can always check for dependence using the hasIndependentCopula() method. See [http://openturns.github.io/openturns/latest/user_manual/_generated/openturns.Distribution.html?highlight=distribution#openturns.Distribution.hasIndependentCopula](url)
Thanks for the initiative

**Comment by jonathf** [comment]
Thanks for the quick response, guys!

**Pull request merged**